internal/osutils/autostart/autostart_linux.go

```diff
@@ -16,6 +16,25 @@ const (
 	autostartDir = ".config/autostart"
 )
 
+var data = map[AppName]options{
+	Tray: {
+		launchFileName: constants.TrayAppName,
```

Comment:
I like this use of struct initialization. It's clear to see what you're assigning the constants to.